std/node/_fs/_fs_open.ts

```diff
@@ -0,0 +1,172 @@
+import { existsSync } from "../../fs/mod.ts";
+import { fromFileUrl } from "../path.ts";
+
+type openFlags =
+  | "a"
+  | "ax"
+  | "a+"
+  | "ax+"
+  // | "as"
+  // | "as+"
```

Comment:
Why are those commented out?

Reply:
well, in the [node docs](https://nodejs.org/api/fs.html#fs_file_system_flags)
it says ``'as': Open file for appending in synchronous mode. The file is created if it does not exist.``
i don't know if using "as" and "as+" flags should make fs.open synchronous or not?
but i tested in node and it's not sync

Reply:
i can add them if you want

Reply:
> Why are those commented out? 

well, in the [node docs](https://nodejs.org/api/fs.html#fs_file_system_flags)
it says ``'as': Open file for appending in synchronous mode. The file is created if it does not exist.``
i don't know if using "as" and "as+" flags should make fs.open synchronous or not?